rio_tiler/io/xarray.py

```diff
@@ -232,7 +232,7 @@ def tile(
             reproject_method = resampling_method
 
         if not self.tile_exists(tile_x, tile_y, tile_z):
-            raise TileOutsideBounds(f"Tile {tile_z}/{tile_x}/{tile_y} is outside bounds")
+            raise TileOutsideBounds(f"Tile {tile_x}/{tile_y}/{tile_z} is outside bounds")
```

Comment:
maybe we should print something like `Tile(x={tile_x}, y={tile_y}, z={tile_z})`

Reply:
would you also be able to update https://github.com/cogeotiff/rio-tiler/blob/main/rio_tiler/io/rasterio.py#L344-L347, https://github.com/cogeotiff/rio-tiler/blob/91095a27e663c76f2dcfdac64d4364ad9f9b9585/rio_tiler/io/base.py#L1037-L1040 and https://github.com/cogeotiff/rio-tiler/blob/91095a27e663c76f2dcfdac64d4364ad9f9b9585/rio_tiler/io/base.py#L481-L484 🙏